torchgeo/datamodules/digital_typhoon.py

```diff
@@ -0,0 +1,104 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""Digital Typhoon Data Module."""
+
+import copy
+from typing import Any, ClassVar
+
+from torch.utils.data import Subset
+
+from ..datasets import DigitalTyphoonAnalysis
+from ..datasets.digital_typhoon import _SampleSequenceDict
+from .geo import NonGeoDataModule
+from .utils import group_shuffle_split
+
+
+class DigitalTyphoonAnalysisDataModule(NonGeoDataModule):
+    """Digital Typhoon Analysis Data Module."""
+
+    valid_split_types: ClassVar[list[str]] = ['time', 'typhoon_id']
```

Comment:
```suggestion
    valid_split_types = ('time', 'typhoon_id')
```
No need to use a mutable type, tuples are faster and more memory efficient.